hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -219,7 +212,7 @@ Default: 1  (0 in versions prior to 2020-09-03)
 Version: 2020-04-16  
 
 ### FREEZE_VERSION
-Previously this property was only checked by the rpi-eeprom-update script. However, now that self-update is enabled the bootloader will also check this property. If set, this overrides `ENABLE_SELF_UPDATE` to stop automatic updates. To disable `FREEZE_VERSION` you will have to use an SD card boot with recovery.bin.
+Previously this property was only checked by the `rpi-eeprom-update` script. However, now that self-update is enabled the bootloader will also check this property. If set to 1, this overrides `ENABLE_SELF_UPDATE` to stop automatic updates. To disable `FREEZE_VERSION` you will have to use an SD card boot with recovery.bin.
 
 Default: 0  
 Version: 2020-09-03  
```

Comment:
I think this should be `Version: All` ?
Is it worth adding the "Custom EEPROM update scripts must also check for this flag." warning from the previous section? (or is that a non-issue, since nobody ever made any custom EEPROM update scripts?)

Reply:
Changed to All although it's not strictly true because it depends on whether this is referring to the EEPROM or rpi-eeprom-update.
However, almost nobody uses this flag especially now that we have write_protect so let's keep this simple